izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java

```diff
@@ -240,13 +247,16 @@ private void initMavenProperties(PropertyManager propertyManager)
                 } else {
                     value = properties.getProperty(propertyName);
                 }
-                if (propertyManager.addProperty(propertyName, value))
-                {
-                    getLog().debug("Maven property added: " + propertyName + "=" + value);
-                }
-                else
+                if (!propertyName.contains("password") || includePasswordProperties)
```

Comment:
There might be others like `passphrase` that will not be skipped here. Providing a regex check could be more flexible here.
But it matches the requirement of the current issue.

Reply:
I don't like this kind of static substring paring, neither. Should be solved differently, for instance by XML attributes.